DependencyInjection/CmfSeoExtension.php

```diff
@@ -250,20 +251,34 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
     {
         $loader->load('sitemap.xml');
 
-        $container->setParameter($this->getAlias().'.sitemap.configurations', $config['configurations']);
-
+        $configurations = $config['configurations'];
         foreach ($config['configurations'] as $key => $configuration) {
-            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
+//            $container->setParameter($this->getAlias().'.sitemap.'.$key.'_configuration', $configuration);
```

Comment:
well, we are not using the parameter but only the configurations. i
prefer not to clutter the container with unused things. if the need for
it comes up, we can still add it at that point